openassessment/templates/openassessmentblock/message/oa_message_closed.html

```diff
@@ -1,21 +1,15 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">
-        {% if not_yet_open %}
-            {% trans "This Task Is Not Yet Available" %}
-        {% else %}
-            {% trans "This Assignment Has Closed" %}
-        {% endif %}
-    </h3>
+    <h5 class="message__title">{% trans "Status" %}</h5>
```

Comment:
@staubina I agree with @cahrens here. I think if its the status of the entire peer assessment it should be an H4.  If its the status of one of the individual steps, it should be an H5.